database.go

```diff
@@ -475,6 +475,8 @@ func (m *Measurement) walkWhereForSeriesIds(expr influxql.Expr, filters map[uint
 
 			// finally return the ids and say that we should include them
 			return ids, true, nil
+		default:
```

Comment:
This may not be the right thing to do, but it would have flagged the bug immediately.